### ISSUE TITLE
Documentation of CHGate incorrect

### ISSUE THREAD

**Issue by daniel-miller-quantum** [Qiskit/qiskit-metapackage#1053]
### Informations

- **Qiskit version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?
There is a minor error in the documentation; It is not possible to factor out the 1/\sqrt(2) like this because then the upper left diagonal entries are 1/sqrt(2) instead of 1.

### Steps to reproduce the problem
Go on https://qiskit.org/documentation/stubs/qiskit.circuit.library.CHGate.html#qiskit.circuit.library.CHGate

Look for the box "!note" which starts with "In Qiskit’s convention, higher qubit indices are more significant (little endian convention). In many textbooks, controlled gates are presented with the assumption of more significant qubits as control, which in our case would be q_1. Thus a textbook matrix for this gate will be:"

### What is the expected behavior? 
You see the following:
![image](https://user-images.githubusercontent.com/72351187/95081772-5619af00-071a-11eb-8791-a0780b579a62.png)

### Suggested solutions
Put the 1/\sqrt(2) entrywise into the bottom-right block of the matrix.

**Comment by daniel-miller-quantum** [comment]
Sorry, this is not a bug but a documentation issue.

**Comment by g-ebbs** [comment]
I looked into this issue and I believe I fixed the code in the docs to match the suggested solution. 

Here is what the solution looks like:
![image](https://user-images.githubusercontent.com/59269494/95684684-5950e700-0bc1-11eb-8cb2-f337c5a4bba3.png)

This is my first time submitting an issue fix so I want to make sure I have done everything correctly. Could someone help me properly test and submit the pull request? 
Hope this helps and I'm excited to continue contributing!
